scripts/release-commits.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I tested this logic here: 
https://github.com/Integralist/actions-testing/runs/4984447215?check_suite_focus=true